src/utils/parseQueryParams.js

```diff
@@ -0,0 +1,46 @@
+export function parseQueryParams() {
+  var queryString = window.location.search;
+  if (queryString === '') return {};
+  var keyValPairs = [];
+  var params = {};
+  queryString = queryString.replace("?", "");
+  queryString = queryString.replace(/%20/g, " ");
+  queryString = queryString.replace(/%2C/g, ",")
+
+  if (queryString.length) {
+    keyValPairs = queryString.split('&');
+    for (var pairNum in keyValPairs) {
+      var key = keyValPairs[pairNum].split('=')[0];
+      if (!key.length) continue;
+      if (typeof params[key] === 'undefined') {
+        params[key] = [];
+      }
+
+      var values = keyValPairs[pairNum].split('=')[1];
+      values = values.split(',');
+
+      if (Array.isArray(values) && values.length > 1) {
```

Comment:
Don't need the first `Array.isArray()` check, `values` will always be an array (`.split()` will always return an array)